EpicLoot/EpicLoot.cs

```diff
@@ -232,8 +232,8 @@ private void Awake()
 
             LoadAssets();
 
-            ExtendedItemData.LoadExtendedItemData += MagicItemComponent.OnNewExtendedItemData;
-            ExtendedItemData.NewExtendedItemData += MagicItemComponent.OnNewExtendedItemData;
+            //ExtendedItemData.LoadExtendedItemData += MagicItemComponent.OnNewExtendedItemData;
```

Comment:
I think we can go ahead and remove the commented lines, git will save the history

Reply:
Done.